app/scripts/modules/netflix/fastProperties/scope/fastPropetyScopeSearch.less

```diff
@@ -0,0 +1,148 @@
+@import "../../../core/presentation/less/imports/commonImports.less";
+
+.fp-scope-search {
+  margin-top: 6px;
+  list-style: none;
+  .dropdown-menu {
+    min-width: 206px;
+    top: 33px;
+    z-index: 10000;
+    margin-left: 15px;
+    li {
+      color: #999;
+      max-width: 300px;
+      strong {
+        font-weight: 600;
+      }
+      a, .category-heading {
+        padding-left: 15px;
+        padding-right: 15px;
+      }
+      a {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        font-variant: normal;
+        color: #000;
+        transition: background-color 0.2s, outline-width 0.2s;
+        .expand-results {
+          font-weight: bold;
+        }
+        &.sublinked, &.sublink {
+          display: inline-block;
+          padding-right: 5px;
+        }
+        &.sublink {
+          padding: 3px 4px;
+        }
+      }
+      .sublink-label {
+        display: inline-block;
+        padding: 3px 0;
+        overflow: hidden;
+      }
+      &.category-heading {
+        margin-top: 10px;
+        padding-top: 10px;
+        border-top: 1px solid #f1f1f1;
+        text-transform: uppercase;
+      }
+      &:first-child {
+        margin-top: 0;
+        border: none;
+      }
+    }
+  }
+  .glyphicon-search {
+    pointer-events: initial;
+  }
+}
+
+
+.scope-category-container .category-heading {
+  font-weight: 600;
+  color: @text-color;
+  margin: 10px 0;
+}
+
+
+.scope-button-container {
+  display: inline-flex;
+}
+
+.scope-button-container .badge-impact-count {
+  position: absolute;
+  left: -7px;
+  top: -8px;
+}
+
+.scope-button-container .badge-impact-count.positive {
+  background: @unhealthy_red;
+}
+
+.scope-button-container
+.fast-property-scope-button
+div {
+  margin: 5px 0;
+}
+
+.scope-button-container
+.fast-property-scope-button
+div
+span:not(:first-of-type):before {
```

Comment:
here (and the rule above) - it's a little confusing to read without any indentation or curly brackets